lightningd/options.c

```diff
@@ -694,6 +700,9 @@ static const struct config mainnet_config = {
 
 	/* Allow to define the default behavior of tor services calls*/
 	.use_v3_autotor = true,
+
+	/* 10 minutes should be enough for anyone! */
+	.connection_timeout_secs = 600,
```

Comment:
This is *very* conservative ? I'd rather not start funding a channel with a peer who took 9 minutes to respond to init..